src/Ocelot/WebSockets/ClientWebSocketOptionsProxy.cs

```diff
@@ -12,9 +12,13 @@ public ClientWebSocketOptionsProxy(ClientWebSocketOptions options)
     {
         _real = options;
     }
-
+#if NET7_0_OR_GREATER
     public Version HttpVersion { get => _real.HttpVersion; set => _real.HttpVersion = value; }
     public HttpVersionPolicy HttpVersionPolicy { get => _real.HttpVersionPolicy; set => _real.HttpVersionPolicy = value; }
+#else
+    public Version HttpVersion { get; set; }
+    public HttpVersionPolicy HttpVersionPolicy { get; set; }
+#endif
```

Comment:
Wow! Why did you fallback the logic?
Doesn't NET6 support expression bodies for properties?

Reply:
No, native ClientWebSocketOptions doesn't support HttpVersion and HttpVersionPolicy in .NET 6

Reply:
Emmm... And we cannot remove properties because of interface definitions for .NET 7+
Maybe preprocess `IClientWebSocketOptions` interface to remove them?

Reply:
Let's keep it like that for the minute, I think this code should be reviewed. It's a bit odd. You already wrote something about Websockets somewhere...